designs/2022-community-eslint-org/README.md

```diff
@@ -0,0 +1,390 @@
+Secondly, It could also put pressure on packages from other ESLint related
+organizations/maintainers to transfer their project to this new org, as it could
+be seen as the "correct" place for popular packages.
+
+## Backwards Compatibility Analysis
+
+<!--
+    How does this change affect existing ESLint users? Will any behavior
+    change for them? If so, how are you going to minimize the disruption
+    to existing users?
+-->
+
+It won't affect existing ESLint users, as package could be released under the
+same name on `npm`.  
+Even when we would release a package under the `@eslint-community` `npm` scope
+(for instance because we can't get the rights to publish to the existing `npm`
+package), people could still use the old package without any extra problems
+(except for the ones they already had) if they wanted to.
+
+## Alternatives
+
+<!--
+    What other designs did you consider? Why did you decide against those?
+
+    This section should also include prior art, such as whether similar
+    projects have already implemented a similar feature.
+-->
+
+- Private persons could create a fork of the above mentioned packages.  
+  This could however create several well maintained forks, which would cause
+  fragmentation in the community.
+
+  More importantly, this could cause the "problem" of the fork being reliant on
+  that specific individual, which could put us in the same position
+  (abandoned/unmaintained package) all over again at some point in time.
+  Possibly having a new fork each X amount of time, would also have the burden
+  of needing to update all existing dependants each time.
+
+- The community could independently create multiple more focused organizations
+  to maintain a subset of ESLint related packages.  
+  Some examples of already existing organizations like that are
+  [@import-js](https://github.com/import-js) (which maintains
+  [`eslint-plugin-import`](https://github.com/import-js/eslint-plugin-import)),
+  [@jsx-eslint](https://github.com/jsx-eslint) (which maintains
+  [`eslint-plugin-jsx-a11y`](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y),
+  [`eslint-plugin-react`](https://github.com/jsx-eslint/eslint-plugin-react) &
+  [`jsx-ast-utils`](https://github.com/jsx-eslint/jsx-ast-utils)) & `@standard`
+  (which maintains a couple of
+  [`standard`](https://github.com/standard/standard) related ESLint configs)
+
+## Open Questions
+
+<!--
+    This section is optional, but is suggested for a first draft.
+
+    What parts of this proposal are you unclear about? What do you
+    need to know before you can finalize this RFC?
+
+    List the questions that you'd like reviewers to focus on. When
+    you've received the answers and updated the design to reflect them,
+    you can remove this section.
+-->
+
+- What exact acceptance criteria do we want to use?
+- What's the main place of communication for the community core team?
+- Will the community core team be compensated out of the main ESLint budget?
```

Comment:
@nzakas I'm sure all of the above mentioned people who want to take the lead in this aren't doing it for the money.

I can only speak for myself (although I think most of us are thinking kind of the same way on this), so for me personally, I would do all the "work" regardless if I'm being paid or not.
Having some way of appreciation/compensation for spending time on this & running day to day activities would be a welcome extra, but it isn't my main motivator for doing this.

Reply:
Well, let’s not fool ourselves, this wouldn’t be full-time money. :)

I was more thinking we could pay the same hourly rate we do for the team, not necessarily to motivate but rather to show appreciation for taking the time to do it. People’s time is worth something, and if folks are putting in a decent amount of work to benefit the ESLint project, then I think that deserves payment just like on other parts of the project.

Reply:
I've added a note about the ESLint TSC Meeting
Once the transcript is merged, the link will work.